File: sol/solution/0001-0100/0026. Remove Duplicates from Sorted Array/Solution.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        if (nums == null || nums.Length == 0) {
            return 0;
        }
        int write = 1;
        for (int read = 1; read < nums.Length; read++) {
            if (nums[read] != nums[read - 1]) {
                nums[write++] = nums[read];
            }
        }
        return write;
    }
}

File: sol/solution/0001-0100/0026. Remove Duplicates from Sorted Array/Solution.kt
class Solution {
    fun removeDuplicates(nums: IntArray): Int {
        if (nums.isEmpty()) return 0
        var write = 1
        for (i in 1 until nums.size) {
            if (nums[i] != nums[write - 1]) {
                nums[write] = nums[i]
                write++
            }
        }
        return write
    }
}